git_machete/client.py

```diff
@@ -219,7 +219,8 @@ def add(self,
             opt_onto: Optional[LocalBranchShortName],
             opt_as_root: bool,
             opt_yes: bool,
-            verbose: bool = True
+            verbose: bool = True,
```

Comment:
While you're here, I'd avoid default params here, let's make the caller always decide explicitly what they want